toolbox/bin/variables.sh

```diff
@@ -58,7 +58,11 @@ fi
 # populate current layer info
 LAYERS_DIR="$WORKING_DIR/layers"
 LAYERS=''
-if [ -d "$LAYERS_DIR" ]; then
+if [ ! -d "$LAYERS_DIR" ]; then
+  # CircleCI fails the build is none of the files mentioned in persist_to_workspace exist.
+  # changed_layers is the only one that makes to create in this situation
```

Comment:
I'm not sure what you meant to say here?

Reply:
changed phrasing there